Description: Use FHS compliant paths by default
Author: Oliver Korff <ok@xynyx.de>
Last-Update: 2011-10-13

--- a/option.c
+++ b/option.c
@@ -1093,9 +1093,9 @@
     char *readstat = (char *) -1;
     int lines = 0;
 
-    helpfile = fopen("crafty.hlp", "r");
+    helpfile = fopen("/usr/share/doc/crafty/crafty.hlp", "r");
     if (!helpfile) {
-      printf("ERROR.  Unable to open \"crafty.hlp\" -- help unavailable\n");
+      printf("ERROR.  Unable to open \"/usr/share/doc/crafty/crafty.hlp\" -- help unavailable\n");
       return (1);
     }
     if (nargs > 1) {
